src/easylink/pipeline_schema_constants/testing.py

```diff
@@ -582,3 +582,55 @@
     ),
 ]
 SCHEMA_PARAMS_EP_HIERARCHICAL_STEP = (NODES_EP_HIERARCHICAL_STEP, EDGES_ONE_STEP_TWO_ISLOTS)
+
+NODES_OUTPUT_DIR = [
```

Comment:
I hijacked @zmbc 's failing test commit that highlighted the no-directory-support issue